lib/newgistics/update_shipment.rb

```diff
@@ -0,0 +1,27 @@
+module Newgistics
+  class UpdateShipment
+    include Virtus.model
+
+    attribute :id, String
+    attribute :order_id, String
+    attribute :add_items, Array[Item]
+    attribute :remove_items, Array[Item]
+    attribute :success, Boolean
+
+    attribute :errors, Array[String], default: []
+    attribute :warnings, Array[String], default: []
+
+    def success?
+      !!success
+    end
+
+    def update
```

Comment:
If we go with the `Newgitics::ShipmentUpdate` name I think we could rename this to `save` to have a similar API with `Newgistics::Order` also I think it fits in nicely:

```ruby
shipment_update = Newgistics::ShipmentUpdate.new(attributes)
shipment_update.save
```